content/sql/glossaries/operators/like.md

```diff
@@ -0,0 +1,120 @@
+---
+Title: "SQL Operators: `LIKE`"
+Subjects:
+  - "Language Fluency"
+  - "Code Foundations"
+  - "Data Science"
+Tags:
+  - "Database"
```

Comment:
Add "Operators" first.

Take out Tables and add "SQLite",  "MySQL", and "PostgreSQL".